### ISSUE TITLE
This doesn't seem to work properly

### ISSUE THREAD

**Issue by TaoTaoTaos** [mikejongbloet/obsidian-get-stock-information#4]
Sorry, I'm not sure if it's my setup error or if this plugin has stopped working. It seems like it's not working properly.

**Comment by lukehowlett97** [comment]
Same here. Just says failed when I try

**Comment by daveg7lee** [comment]
same here.

**Comment by Rad88nok** [comment]
Yup,  for XTB.WA (https://finance.yahoo.com/quote/XTB.WA/) Don't work

**Comment by OPEN1004MIND** [comment]
me too 
Error occurred: Cannot convert undefined or null to object
Can't load information

**Comment by jhssilva** [comment]
There will be a fix on this?
